### ISSUE TITLE
Locales

### ISSUE THREAD

**Issue by gregoriopellegrino** [sc0ttkclark/exports-and-reports#9]
Hi,
I think your plugin is great. One really useful thing would be manage locales, for example in Italy we use commas as decimal separators and the date format is dd/MM/yyyy.

Thanks,
Gregorio

**Comment by sc0ttkclark** [comment]
I agree here, but it will probably have to wait for the big rewrite where I revamp the UI. There's a lot of stuff that has to happen to support more UI options.
